### PR TITLE
Updated resource tier units in output

### PR DESCRIPTION
**Pull request by stingare**
Updated units:
```
------------------------------------------
--- ResourceTier 200 - Dynamic Compute ---
------------------------------------------
Maximum CPU time during initialization: 30 ms
Maximum wall time during initialization: 100 ms
Maximum memory usage per event handler: 1.5 MB
Maximum CPU time per event handler: 10 ms
Maximum wall time per event handler: 4 s
Maximum number of HTTP sub-requests allowed from a parent request: 1
Maximum number of HTTP sub-requests allowed in parallel per request: 1
Maximum wall time per HTTP sub-request: 1 s
Maximum response size per HTTP sub-request: 5 MB
Maximum memory usage for responseProvider: 2 MB
Maximum CPU time for responseProvider: 100 ms
Maximum wall time for responseProvider: 4 s
Maximum number of HTTP sub-requests allowed for responseProvider: 50
Maximum number of HTTP sub-requests allowed in parallel for responseProvider: 5
Maximum wall time for HTTP sub-requests during the execution of the responseProvider event handler: 1 s
Maximum response size for HTTP sub-requests during the execution of the responseProvider event handler: 5 MB
```

### CODE REVIEW

File: src/utils/cli-utils.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import * as edgeWorkersClientSvc from '../edgeworkers/client-manager';
 import * as envUtils from '../utils/env-utils';
+
 const inquirer = require('inquirer');
 const Spinner = require('cli-spinner').Spinner;
+const TIME_UNITS: string[] = ['ms', 's', 'min', 'h'];
+const MEMORY_UNITS: string[] = ['B', 'kB', 'MB', 'GB', 'TB'];
+const COUNT_UNITS: string[] = ['', 'Thousand', 'Million', 'Billion', 'Trillion'];
```
Comment (swathimr) [inline src/utils/cli-utils.ts:8]: what does empty string mean here ?

File: cli.json [hunk 2]
```diff
@@ -5,13 +5,13 @@
   "commands": [
     {
       "name": "edgeworkers",
-      "version": "1.4.0",
+      "version": "1.4.2",
```
[the remark this answers is not in the document]
Comment (stingare) [reply cli.json]: Updated to 1.4.1

File: src/utils/cli-utils.ts (hunk 1 above)
Comment (stingare) [reply src/utils/cli-utils.ts:8]: Empty string is the count limit unit for when the count value is under 1000 (i.e 1-999 count will have the empty string, and 1000 would be displayed as 1 Thousand)

**Review by swathimr — APPROVED** [review]
LGTM

**Pull request merged**